src/main/java/uk/gov/ons/census/action/builders/PrintFileDtoBuilder.java

```diff
@@ -17,7 +17,8 @@
           ActionType.P_RL_2RL1A,
           ActionType.P_RL_2RL2BA,
           ActionType.P_RL_1IRL1,
```

Comment:
`P_RL_1IRL1` - this one shouldn't be on the list. It must have been added by somebody else by accident